develop/components/pages/TablePage.tsx

```diff
@@ -206,6 +216,17 @@ const TablePage: React.FunctionComponent = () => {
                         <Table columns={columns} data={smallData} onRowSelected={(rows: Array<TableRow>) => {}} />
                     </div>
 
+                    <p>Here is an example with inline edit</p>
+                    <div className="result wide">
+                        <div className="row">
+                            <div className="col text-right">
+                                <Button title="Cancel" label="Cancel" disabled={!editMode} onClick={() => setEditMode("cancel")} className="mr-2" />
+                                <Button title="Update" label={editMode === "edit" ? "Save" : "Edit"} onClick={() => setEditMode(editMode === "edit" ? "save" : "edit")} />
```

Comment:
label={editMode === "edit" ? "Save" : "Edit"} 
```suggestion
                                <Button title="Update" label={editMode === "edit" ? "Edit" : "Save"} onClick={() => setEditMode(editMode === "edit" ? "save" : "edit")} />
```

Reply:
No, the current one is correct omar. Edit comes before save, the default button text is edit